airbrushedairbrushed---
layout: post
title: Moorea 2022 Coral Spawning and Field Expedition Daily Entries  
date: '2022-09-18'
categories: Moorea_Coral_Spawning_2022
tags: Fieldwork Gametogenesis Sampling Physiology Reproduction Spawning
---

This post includes the daily entry notebook for Sept-Oct 2022 *Acropora* coral spawning and field expedition in Moorea, French Polynesia. These notebook posts are updated in order of most recent to oldest.  

# Project Overview  

The goal of this field expedition is to conduct spawning research in September and October in *Acropora* spp. Further, we will be continuing the monthly gametogenesis timeseries, sampling *Acropora pulchra* and *Pocillopora spp.* throughout gametogenesis on the northshore backreef to collect data on histology, physiology, and epigenomics. Original tagging and sampling of 40 *Acropora pulchra* colonies was completed by the Putnam Lab team (Hollie, Ariana, Pierrick, and Alex (CRIOBE)) in December 2021 and of *Pocillopora spp.* starting in January 2022. Monthly timepoint sampling will be carried out for this project until October 2022 for spawning. We will be processing physiological measurements (symbiont densities, chlorophyll content, total protein, surface area, and tissue biomass) for collected and preserved samples, measuring bleaching score on *in situ* colonies, decalcifying histological samples, and collecting molecular and histological samples during our three month trip.

A. Huffmyer is conducting field work to investigate metabolic integration of coral host and symbiont in early life history as part of funded National Science Foundation Ocean Sciences Postdoctoral Fellowship and the University of Washington eScience Data Science Postdoctoral Fellowship. More information on the abstract of this award can be [found here](https://www.nsf.gov/awardsearch/showAward?AWD_ID=2205966&HistoricalAwards=false). Ariana's notebook can be [viewed here](https://ahuffmyer.github.io/ASH_Putnam_Lab_Notebook/Moorea-2022-Coral-Spawning-and-Field-Expedition-Daily-Entry/).  

D. Becker Polinksi is conducting field work to characterize maternal provisioning, epigenetic inheritance, and cross-generational impacts of thermal stress on *Acropora pulchra* throughout gametogenesis and early development (i.e., embryo, larval development, recruits, and juveniles) with parents that have previously experienced a marine heatwave event in March-April 2022. This research will connect molecular mechanisms impacts on phenotypic response through gametogenesis, larval development, and post settlement, to help understand acclimatization patterns facilitated by parental inheritance. All data and further information can be found on her personal GitHub [here](https://github.com/daniellembecker/Gametogenesis).

We will also continue in situ monitoring of *Acropora* and *Pocillopora* colonies on the reef in Moorea.   

Finally, we will conduct an experiment to expose adult *Acropora pulchra* corals to conditions that elicit changes in symbiotic and metabolic condition and test how these changes influence response to a repeat stressor. We are manipulating light conditions to elicit metabolic and symbiotic responses and testing stress response at the end of the experiment using the thermal stress CBASS system. Notebook post detailing activities specific to the E5 adult metabolism and stressor experiment (details below) [can be found here](https://urol-e5.github.io/2022/09/20/E5-Moorea-Adult-Stressor-Experiment.html) and on the [E5 A. pulchar Metabolism GitHub repository](https://github.com/urol-e5/apulchra_metabolism).   

*Field expedition members:*     
Ariana Huffmyer  
Pierrick Harnay  
Danielle Becker-Polinski  

# 25 September 2022  

### *Logger calibration*  

Ariana removed loggers from tanks at 1030 and downloaded all logger files at 1200 and began writing a script to calulate the offsets for each logger.  

### *Tank set up*  

Ariana set up the E5 tanks to have hanging wires for each fragment when fragmentation occurs tomorrow.  

![tanks 1](https://ahuffmyer.github.io/ASH_Putnam_Lab_Notebook/images/NotebookImages/Moorea2022/e5_tank1.jpeg)

![tanks 2](https://ahuffmyer.github.io/ASH_Putnam_Lab_Notebook/images/NotebookImages/Moorea2022/e5_tank2.jpeg)

# 24 September 2022  

### *Airbrushing gametogenesis project - September 2022*

Danielle taught Pierrick the [airbrushing protocol](https://github.com/daniellembecker/Gametogenesis/blob/main/protocols/2020-01-01-Airbrushing.md) for Mo'orea and also airbrushed 40 of the September 2022 monthly sampling *A. pulchra* fragments following the protocol and stored the tissue homogenate in the Molecular -40 freezer in a mesh bag. 

### *Logger calibration*  

Ariana added the two pendant loggers from the incubators into the calibration tank that were missed yesterday.  

Logger 21335989 - logging every 10 min  
Logger 21002976 - logging every 10 min  

Added to calibration tank at 0900. Note that data from 0700-0830 may need to be removed from the calibration. At this time the morning sun is causing shadows in the tank that may make the calibration in accurate.  

### *Daily measurement probe set up*  

Daily measurements for larval and the E5 adult project should include:  

1. **Flow rate (mL / 10 sec)**: Use a graduated clyinder to measure the amount of water from the input tube in a 10 sec period.    
2. **Temperature (C)**: Measurement of temperature in the center of each tank.   
3. **Light (PAR)**: Measurement of light at the center of the tank and in the center of each of 4 quadrants (total of 5 measurements per tank).    
4. **pH and salinity**: Measure pH and salinity with the Orion probe at the center of each tank.  

Time of day, tank number, and notes regarding weather or other conditions should be noted.  

For Ariana's larval project, daily measurements will be recorded in [Google Drive here](https://docs.google.com/spreadsheets/u/1/d/1wYqONrAvGfsWG7UNqLmtOYiIFESSJR6olzfFf_IhgYg/edit?usp=drive_web&ouid=110285914417149495351).   

For the E5 adult project, daily pH, temperature, and flow measurements will be recorded in [GitHub here](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/environmental/daily_measurements/daily_pH_temp_flow.xlsx) and daily light measurements will be recorded in [GitHub here](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/environmental/daily_measurements/daily_light.xlsx).  

The protocol for taking daily measurements was written by Ariana available here in the [E5 protocols repository](https://github.com/urol-e5/protocols/blob/master/2022-09-24-Moorea-Daily-Measurement-Protocol.md) based on the [Putnam Lab protocol](https://github.com/Putnam-Lab/Lab_Management/blob/master/Lab_Resources/GSO_Wetlab_Protocols/GSO_Wetlab_Protocols.md#II-Tris-Calibration).  

Ariana prepared all probes and measured a standard curve for the pH probe across a range of temperature with Tris using the [Putnam Lab protocol](https://github.com/Putnam-Lab/Lab_Management/blob/master/Lab_Resources/GSO_Wetlab_Protocols/GSO_Wetlab_Protocols.md#II-Tris-Calibration). 15 mL of Tris was added to a falcon tube and placed in the fridge for ~20 min. The temperature and pH probe were then submerged into the falcon tube on the bench top. The calibration started at 11°C and slowly warmed to room temperature with measurements of temperature and Tris every ~1°C. Once the Tris reached room temperature the Tris and probes were moved into a benchtop incubator (MyTemp Mini) and warmed to the upper temperature used in the calibration (~35C).  

This data is below and was recorded in Ariana's [Google Drive here](https://docs.google.com/spreadsheets/d/1wYqONrAvGfsWG7UNqLmtOYiIFESSJR6olzfFf_IhgYg/edit#gid=1109057105) and in the [E5 repository here](https://github.com/urol-e5/apulchra_metabolism/blob/main/data/environmental/20220924_Tris_Calibration.csv).  

| Date     | Time | Temp.Tris | mV.Tris |
|----------|------|-----------|---------|
| 20220924 | 1013 |     11.18 |     -73 |
| 20220924 | 1016 |     13.31 |     -72 |
| 20220924 | 1020 |     14.66 |   -70.3 |
| 20220924 | 1025 |     16.18 |   -68.3 |
| 20220924 | 1030 |     17.28 |   -66.7 |
| 20220924 | 1049 |     18.75 |   -64.9 |
| 20220924 | 1057 |     19.48 |   -64.1 |
| 20220924 | 1102 |     20.18 |   -63.1 |
| 20220924 | 1106 |     21.08 |   -61.9 |
| 20220924 | 1111 |     22.19 |   -60.6 |
| 20220924 | 1115 |     23.07 |   -59.4 |
| 20220924 | 1120 |     24.04 |     -58 |
| 20220924 | 1125 |     25.03 |   -56.7 |
| 20220924 | 1130 |     26.06 |   -55.4 |
| 20220924 | 1135 |     27.27 |   -53.9 |
| 20220924 | 1140 |        28 |   -52.8 |
| 20220924 | 1145 |     28.96 |   -51.8 |
| 20220924 | 1150 |     30.42 |   -49.6 |
| 20220924 | 1153 |     31.06 |   -48.6 |
| 20220924 | 1159 |     32.27 |   -47.1 |
| 20220924 | 1204 |     33.35 |   -45.6 |
| 20220924 | 1210 |        34 |   -44.8 |
| 20220924 | 1215 |     35.12 |   -43.1 |
| 20220924 | 1220 |     36.22 |   -41.6 |

# 23 September 2022  

### *Heatwave colony return*  

The team returned the heatwave *A. pulchra* colonies from Danielle's project to the Mahana site from 0900-1100. We will collect them again before the next spawning. Colonies were zip tied to cinder blocks on Patch 3 at Mahana.  

![adult return](https://ahuffmyer.github.io/ASH_Putnam_Lab_Notebook/images/NotebookImages/Moorea2022/adultreturn.jpeg)   

### *Adult E5 experiment set up*  

Ariana cleaned water lines and started water in n=6 tanks in preparation for the adult experiment.  

Ariana started a [GitHub repo for this project here](https://github.com/urol-e5/apulchra_metabolism).  

### *Logger offload and calibration*  

Loggers were read off for the wildtype parent tanks and Danielle's parent tanks. Files were labeled with SSN and date and saved in the Apulchra_metabolism E5  GitHub repository. Loggers read off were:

20719656 (Wildtype Tank 2)  
20946644 (Danielle Tank 1)    
20444033 (Wildtype Tank 1)   
20444040 (Danielle Tank 2)  

Ariana started a calibration of all temperature and light loggers that will run over the next 24-48 h. This will allow us to convert Lux to PAR from the Hobo Pendant loggers and to calculate temperature offsets from the temperature logging on the Hobo Pendants.  

The following loggers were launched:  

| Logger   | Interval | Type          | Time Deployed  |
|----------|----------|---------------|----------------|
| 20719656 | 10 min   | Hobo Temp     | 1635           |
| 20946644 | 10 min   | Hobo Temp     | 1635           |
| 20444033 | 10 min   | Hobo Temp     | 1635           |
| 20444040 | 10 min   | Hobo Temp     | 1635           |
| 21335986 | 10 min   | Pendant       | 1635           |
| 21002981 | 10 min   | Pendant       | 1635           |
| 21002972 | 10 min   | Pendant       | 1635           |
| 20937872 | 10 min   | Pendant       | 1635           |
| 21002975 | 10 min   | Pendant       | 1635           |
| 21335980 | 10 min   | Pendant       | 1635           |
| 21335987 | 10 min   | Pendant       | 1635           |
| 21002980 | 10 min   | Pendant       | 1635           |
| 21002977 | 10 min   | Pendant       | 1635           |
| 21002974 | 10 min   | Pendant       | 1635           |
| 21335982 | 10 min   | Pendant       | 1635           |
| 21002982 | 10 min   | Pendant       | 1635           |
| 20937873 | 10 min   | Pendant       | 1635           |
| 21335983 | 10 min   | Pendant       | 1635           |
| 21335984 | 10 min   | Pendant       | 1635           |
| 21002979 | 10 min   | Pendant       | 1635           |
| 15643    | 10 min   | Odyssey Light | 1635           


The Odyssey logger was previously calibrated by Danielle Becker-Polinski [with this protocol](https://daniellembecker.github.io/DanielleBecker_Lab_Notebook/Odyssey-PAR-Loggers-Protocol/).  

Loggers were arranged as pictured below in one tank with running water at a low flow to avoid surface disturbance. These loggers will run for 24-48 h to conduct the calibration for light and temperature data.  

![loggers](https://ahuffmyer.github.io/ASH_Putnam_Lab_Notebook/images/NotebookImages/Moorea2022/loggers.jpeg)   

# 22 September 2022  

### *Histology time series*   

We completed a sampling time point for the *A. pulchra* histology time series at the Mahana site (see details beow from 16 September 2022) for the high frequencing sampling. The purpose of this sampling is to acquire higher resolution histology from this species through the last month until spawning. This will occur every 5 days.  

We sampled n=20 colonies (n=4 from each patch). We sampled between 0930 and 1100. We returned to the lab and processed the samples as described on 17 September 2022 between 1100 and 1230.  

### *E5 project set up*   

Ariana continued setting up materials for the E5 adult stressor project. We purchased 6 bins that will be tanks for this experiment. Ariana prepared the bins by drilling drain holes and securing wire at the top of each tank to hang fragments.  

### *Tetiaroa spawning preparations*  

The team met with collaborators in the Padilla-Gamino lab from the University of Washington to assist with preparations for a collaborative spawning expedition that will occur in October. The Padilla-Gamino lab will conduct spawning in Tetiaroa in October with parallel measurements in Moorea to characterize reproduction in *Acropora* corals. We shared protocols with their lab for spawning and measuring response variables of interest.  

### *Gamete dissection*  

Danielle trialed methods for dissecting gametes from live *A. pulchra* fragments. During our field expedition today, we noticed that gametes were visible in *A. pulchra* fragments and Danielle dissected and sampled these gametes.  

Danielle first cut extra fragments that were taken from sampling with bon cutters into a ~0.5 inch piece and submerged in a microscope dish under a microscope light to identify if any gonads were present in the fragment piece. She then used fine tipped tweezers to hold the fragment piece and used a fine pointed dissecting tool to gently dislodge any of the gonads from the underlying tissues. She then used a glass pipette to apply pressure and suction to the gonad until it finally released from the tissue. She then preserved the 9 gonads she was able to collect from 5 separate fragments into 250 uL of DNA/RNA shield in a sterile o-ring micro-centrifuge tube. This process will be modified next sampling period to try a fragment saw and needle syringe to improve the number of gonads extracted from samples. The below images show clear gonads (small white circular material in the skeleton)located in the polyp pockets of the fragment pieces.

![image1](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/gonad_dissection.jpeg)

![dissectingimage](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/gonad_dissection2.jpeg)

![fragment piece](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/gonad_piece.jpeg)


# 21 September 2022  

### *Project set up*  

Ariana, Pierrick, and Danielle sourced materials for the E5 adult project including buying tank bins, finding shade cloth, and cleaning water tables. Light levels in the high light treatment will be 500-700 PAR peaking during the day and the low light treatment will reduce by 65% to approx. 100-300 PAR. [See details on this project here](https://urol-e5.github.io/2022/09/20/E5-Moorea-Adult-Stressor-Experiment.html).  

Ariana also discussed experimental design with the E5 team and we have revised the experimental approach to include two sampling time points with fragments sourced from 10 colonies/genotypes.

Ariana will finish set up for this project this week and then move forward with fragging and starting the experiment once design is finalized.

### *A. pulchra high frequency sampling preparation*  

Danielle selected the bags and labeled tubes for the subset of coral colonies and tags that will be samples for the high frequency project. Bags and tubes for high frequency sampling will be labeled with HF colony ID (Ex: HF 416)

Metadata:

| Species    | Site   | OrderbyPatch | Original.Tag.Num | Patch | Latitude.S | Longitude.W | GPS.Point.Name | Timestamp |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
|------------|--------|--------------|------------------|-------|------------|-------------|----------------|-----------|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|---|
| A. pulchra | Mahana | P1.3         |              414 | P1    | 17°29.234' | 149°53.264' | P1             |  12:59 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P2.5         |              416 | P2    | 17°29.232' | 149°53.253' | P2             |   1:11 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P3.4         |              424 | P3    | 17°29.223' | 149°53.249' | P3             |   1:17 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P5.1         |              426 | P5    | 17°29.220' | 149°53.236' | P5             |   1:49 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P2.1         |              429 | P2    | 17°29.232' | 149°53.253' | P2             |   1:05 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P2.4         |              430 | P2    | 17°29.232' | 149°53.253' | P2             |   1:10 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P4.9         |              431 | P4    | 17°29.231' | 149°53.242' | P4             |   1:46 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P1.5         |              433 | P1    | 17°29.234' | 149°53.264' | P1             |   1:02 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P1.4         |              434 | P1    | 17°29.234' | 149°53.264' | P1             |   1:00 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P3.10        |              436 | P3    | 17°29.223' | 149°53.249' | P3             |   1:29 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P5.2         |              437 | P5    | 17°29.220' | 149°53.236' | P5             |   1:50 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P4.10        |              439 | P4    | 17°29.231' | 149°53.242' | P4             |   1:47 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P4.1         |              448 | P4    | 17°29.231' | 149°53.242' | P4             |   1:35 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P2.2         |              449 | P2    | 17°29.232' | 149°53.253' | P2             |   1:07 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P5.8         |              451 | P5    | 17°29.220' | 149°53.236' | P5             |   2:00 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P5.6         |              458 | P5    | 17°29.220' | 149°53.236' | P5             |   1:56 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P3.11        |              461 | P3    | 17°29.223' | 149°53.249' | P3             |   1:30 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P1.1         |              462 | P1    | 17°29.234' | 149°53.264' | P1             |  12:56 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P4.6         |              464 | P4    | 17°29.231' | 149°53.242' | P4             |   1:41 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |
| A. pulchra | Mahana | P3.6         |              465 | P3    | 17°29.223' | 149°53.249' | P3             |   1:22 PM |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |   |

### *Boating*  

Danielle gave the team an orientation to driving the LTER boat (509) at Gump. Ariana took 4 trips as captain driving in the channels to learn navigation on the north shore. Ariana and Pierrick will work on obtaining their authorization for driving Gump boats.

# 20 September 2022  

### *Project planning*  

Ariana developed a draft experimental design for the adult coral E5 repeat stressor experiment. [A separate notebook post has been created here with all details and daily entries for this project in the E5 notebook](https://urol-e5.github.io/2022/09/20/E5-Moorea-Adult-Stressor-Experiment.html). Ariana conducted planning and sourcing materials for this project today.

Overall, this project will expose coral colonies to ambient (450-600 PAR) and reduced (100-200 PAR) light conditions. A draft design is included below. This experiment will run until December. Monthly, fragments will be sampled for molecular, metabolomic, and physiological metrics. In December, S. Matsuda and R. Cunning will measure thermal phenotypes using the CBASS system.  

![E5 project](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/E5project.png)

### *Tank cleaning*  

Ariana cleaned all tanks and larval conicals and stored for use in the next spawning event.  

### *Physiological prep*  

Danielle sent all physiological protocols to Pierrick to prep for physiological processing of the September 2022 monthly sampling of *A. pulchra* starting this week. She also brought all the necessary equipment to the molecular lab for processing and got ziplock bags.

### *A. pulchra high frequency sampling planning*  

Danielle made a new folder for the *A. pulchra* [high frequency histological sampling](https://drive.google.com/drive/u/0/folders/1tctlpdmaiwTbQMSHiTRAfpGczguyVknT) metadata and [schedule](https://docs.google.com/spreadsheets/d/11_jiM5W2MzM6fEoCUPAIJ7kqFw5OB7RjWIYwiTruFec/edit#gid=1170797858) for the next month before spawning. We will be processing a subset of our monthly timepoint *A. pulchra* samples for molecular (snap frozen and DNA/RNA shield) and histology (n=2 ~1 inch pieces of fragment preserved in 4% formalin solution) every five days to categorize and track gametogenesis very close to release. Danielle selected n = 4 from each of her five patches that had intact tags and will be sampling from ~0.5 m above them for the high frequency sampling period.

### *Freezer organization*  

Danielle moved all *A. pulchra* homogenate tubes to biocode -40 freezer and re-labeled a new shelf for Putnam lab. She updated all [Mo'orea freezer/fridge lists](https://docs.google.com/spreadsheets/d/12eTFEpc1uVlKmLpaCqcjT8z_yKTjBXQYZQB8u2bdMqU/edit#gid=1346456213) with new samples. Organized all freezer boxes and freed up space for new samples in molecular freezers.

### *SDR calibration*  

We revised our approach to calibrating the SDR because we need to conduct calibrations at the level of individual wells. We can do this with a user-defined one point adjustment as specified in the SDR user manual and [online resources](https://www.presens.de/support-services/faqs/question/what-is-the-difference-between-user-defined-calibration-and-one-point-adjustment-for-the-sdr-when-do-i-apply-which-one-545).  

The protocl is as follows and **must be completed at the beginning of each measurement set (i.e., at the beginning of each day) at the temperature of measurement**. If the plate will measure multiple temperatures, the one point adjustment will need to be conducted at that temperature. As long as the software remains open, multiple runs can be conducted with one set of calibrations.  

Procedure:  

1. Aerate filtered seawater to generate 100% air saturated seawater (21% oxygen). Allow to fully aerate for 10-15 minutes.  
2. Set the incubators at the desired temperature.
3. Load the SDR plate with the fully aerated seawater in each well ensuring there are no bubbles. Seal the wells with cover slips.
4. Place the plates on SDRs in each incubator. Allow to come to temperature for 10 min.
5. At this point, conduct the one point adjustment.
6. Enter the correct measurement temperature and batch number in the software (PtS6-1624) and salinity.  
7. Select Calibration / One Point Adjustment
8. The calibration will scan all wells.
9. Enter  pO2 (% Air Sat) = 100.
10. Press OK
11. The adjustment has been performed.
12. Repeat at the start of each batch of measurements and at each respective temperature prior to measurement.  

### *Loggers*  

Ariana downloaded all Hobo loggers from the larval tanks to test that loggers are recording data at correct intervals. Tomorrow we will conduct a calibration and offsets for all loggers.  

# 19 September 2022  

### *Project planning*  

Today we talked about project planning for the next few weeks in case the *Acropora pulchra* colonies do not spawn this month. If this is the case, spawning will occur at the following times:  

October Full Moon = Oct 9 10:55am Tahiti time   

*A. pulchra*: 9 nights after the full moon (Oct 18)  
*A. cytherea, A. retusa, A. hyacinthus*: 6-9 nights after the full moon (Oct 15-18)    

We are going to move forward with the following projects.    

1. **Histological time series high frequency sampling**  

Danielle, Ariana, and Pierrick will conduct high frequency histology sampling of the *A. pulchra* histology time series of colonies at Mahana. We will collect samples from a subset of colonies (N=20 colonies, n=4 colonies per patch) every 5 days to characterize the final stages of gametogenesis in October.   

2. **Processing histology time series physiology and histology samples**  

Danielle will train Pierrick how to process physiology and histology samples taken from the *A. pulchra* histology time series in September.  

3. **E5 adult repeat stressor experiment**  

Ariana will plan and start an experiement under the E5 project to conduct a repeat stressor in adult corals. This project will expose fragments of collected wildtype *A. pulchra* to two light treatments. This will be conducted at Gump using bins that will either be covered with shade cloth or under natural light conditions.  

The purpose of this project is to manipulate the metabolic condition of the coral holobiont and understand how metabolic changes influence stress response. A thermal stress test will be conducted in December with Ross Cunning and Shayle Matsuda using the CBASS system. This project will help us understand how metabolic condition influences stress response.   

Throughout the adult exposure, we will characterize physiology, metabolism, and use stable isotope metabolomics to trace symbiotic nutritional exchange.  

4. **October larval symbiont integration experiment**  

Ariana will conduct an experiment in october using the same experimental plan as described for September. Briefly, we will rear *A. hyacinthus* embryos to the larval stage, allow larvae to acquire symbionts, and expose the larvae to either ambient or elevated temperature conditions. Ariana will measure metabolism, survival, and nutritional exchange.

5. **Pocillopora population characterization**  

Pierrick will continue his project characterizing the haplotypes of *Pocillopora* spp. around Moorea. We will conduct additional sampling on transects at the Mahana and Hilton sites in September.

### *Spawning*  

We watched for spawning and no colonies spawned tonight. We will continue watching through Wednesday.  

# 18 September 2022  

### *Setting up temperature control*  

Ariana added heaters to the cooler systems in tanks A and C for high temperature control and connected to the Apex Neptune system. Temperatures programmed to be +4°C to see if the system can give us a high degree of elevated temperature without electrical limitations.  

![heaters](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/heaters.jpeg)  

![heaters2](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/heaters2.jpeg)  

![heaters2](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/tanks2.jpeg)  

The script in the Apex program follows this temperature profile for each virtual stage.  

29C - 30.4C (with 1.5C daily fluctuation)   
Stage 1 = 29  
Stage 2 = 29.2  
Stage 3 = 29.4  
Stage 4 = 29.6  
Stage 5 = 29.8  
Stage 6 = 30.0  
Stage 7 = 30.2  
Stage 8 = 30.4    

This profile will be revised based on the diel fluctuation and the ambient temperature in Moorea. This profile is a test. The peak temperature occurs between 1500 and 1715 daily with this profile.  

The script in the Apex system is as follows:  

```
All master outlets
Set OFF
If Outlet High_Stage1 = ON Then ON
If Outlet High_Stage2 = ON Then ON
If Outlet High_Stage3 = ON Then ON
If Outlet High_Stage4 = ON Then ON
If Outlet High_Stage5 = ON Then ON
If Outlet High_Stage6 = ON Then ON
If Outlet High_Stage7 = ON Then ON
If Outlet High_Stage8 = ON Then ON

High_Stage1
Set OFF
If High_1 < 29 Then ON
If High_1 > 29 Then OFF
If Time 01:00 to 04:30 Then OFF
If Time 06:45 to 00:59 Then OFF

High_Stage2
Set OFF
If High_1 < 29.2 Then ON
If High_1 > 29.2 Then OFF
If Time 01:00 to 03:00 Then OFF
If Time 04:30 to 06:45 Then OFF
If Time 08:15 to 00:59 Then OFF

High_Stage3
Set OFF
If High_1 < 29.4 Then ON
If High_1 > 29.4 Then OFF
If Time 01:00 to 02:00 Then OFF
If Time 03:00 to 08:15 Then OFF
If Time 09:45 to 00:59 Then OFF

High_Stage4
Set OFF
If High_1 < 29.6 Then ON
If High_1 > 29.6 Then OFF
If Time 02:00 to 09:45 Then OFF
If Time 11:15 to 00:30 Then OFF

High_Stage5
Set OFF
If High_1 < 29.8 Then ON
If High_1 > 29.8 Then OFF
If Time 01:00 to 11:15 Then OFF
If Time 12:15 to 23:00 Then OFF
If Time 00:30 to 00:59 Then OFF

High_Stage6
Set OFF
If High_1 < 30 Then ON
If High_1 > 30 Then OFF
If Time 01:00 to 12:15 Then OFF
If Time 13:30 to 19:45 Then OFF
If Time 23:00 to 00:59 Then OFF

High_Stage7
Set OFF
If High_1 < 30.2 Then ON
If High_1 > 30.2 Then OFF
If Time 01:00 to 13:30 Then OFF
If Time 15:00 to 17:15 Then OFF
If Time 19:45 to 00:59 Then OFF

High_Stage8
Set OFF
If High_1 < 30.4 Then ON
If High_1 > 30.4 Then OFF
If Time 01:00 to 15:00 Then OFF
If Time 17:15 to 00:59 Then OFF

```

The temperature treatment was initiated at 1530 today. By 1800 the temperatures were at the desired level. We will let the profiles run for 24 h to view performance. Tomorrow we will look at the Apex data and download loggers.  

### *Setting up SDR and lights*  

Ariana set up the incubators, SDR, and lights that will be required for respirometry and stable isotope incubations.  

![incubators](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/incubators.jpeg)

Aqua Illumination Prime 16HD reef lights are used to provide light in the incubators for photosynthesis measurements and stable isotope incubations.

Light settings were set to 40% intensity on all channels (UV, violet, royal, blue, green, deep red, cool white; no moonlight) positioned 25 cm above surface of the bottom of the incubator and 22 cm above the surface of the SDR plate. The goal is to obtain light at approx. 500 PAR at the level of the SDR plate.  

Light was measured at 6 positions at the level of the SDR plate. This measurement should be conducted at the start of every day of measurement.  

Incubators were labeled Incubator 1 and Incubator 2. Associated SDR 1 and 2 will be placed in the same number incubator.

Light measurements were recorded in Google Drive datasheet. Average light intensity was 494 PAR in Incubator 1 and 487 PAR in Incubator 2.   

A HOBO Onset Pendant logger is placed in each incubator that can be launched at each respiration run to record temperature every 1 min.

Logger 21002976 is in Incubator 1 and logger 2133598 is in Incubator 2.  

![resp1](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/resp1.jpeg)

![resp2](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/resp2.jpeg)

### *Calibrating SDR*  

We replaced oxygen sensor spots on the SDR microplates and need to calibrate since there are new spots from different batches than the old spots. This will also correct any drift of the spots.  

SDR systems were assembled and set up and connected to the PreSens SDR software. To calibrate we followed the following steps:  

1. Mixed solution of 100% oxygen seawater by bubbling filtered seawater and made 0% oxygen seawater by adding sodium sulfite.  
2. Set oxygen unit to % O2 and record every 3 minutes. Select batch PSt5-1624.  
3. Select log measurement to create data for calibration labeled with Date and "calibration". Saved on PC desktop in Moorea 2022 spawning folder. Will be uploaded to Google Drive and GitHub.  
4. Load half of wells with 0% solution and half with 100% oxygen solution in each plate. Record locations. Alternating wells with 0 and 100% water.  
5. Start measurements and let record for 2-3 h until stabilized recording every 3 min.
6. Measure temperature with probe and loggers throughout.  
7. Export the data and calculate mean values for 0 and 100% wells from the last 15 min of measurement. We will create a new calibration dataset from these measurements and upload to the SDR software.   

It is not clear yet how we perform a calibration for each well. We will look into this and try to do a well specific calibration this week.  

### *Spawning*  

We watched for spawning and no colonies spawned tonight. We will continue watching through Wednesday.  

# 17 September 2022  

### *September 2022 Field Day Sampling - Backreef Acropora and Pocillopora*

Danielle, Ariana, and Pierrick conducted monthly timepoint sampling at the Mahana field site (17°29'12.5"S 149°53'16.1"W). Following the [Field Sampling Protocol](https://github.com/daniellembecker/Gametogenesis/blob/main/protocols/2021-12-26-Field-Sampling-Protocol.md), we collected 40 ~5 inch *Acropora pulchra* fragments from the five patches located on the site.  Additionally, 10 4-5 inch fragments of *Pocillopora* were collected and brought back to Gump Station as well for same day processing. For all sampling locations, a close up photo with the tag was taken. Additionally, a bleaching score photo was taken at each sampling location. If a tag was not found, we sampled from the same colony that the tag should have been located on.

After collecting the samples, Danielle, Ariana, and Pierrick followed the [Same Day Sample Processing Protocol](https://github.com/daniellembecker/Gametogenesis/blob/main/protocols/2021-12-26-Sample_Same_Day_Processing_Protocol.md) to prepare samples for histological analysis, molecular and physiological downstream processing.

All bleaching score photos were downloaded off of the Putnam Lab [Canon G7X ](https://www.fnac.com/Compact-Canon-PowerShot-G7X-Mark-II/a9570061/w-4) camera number 2 and uploaded to [Pocillopora_Sampling_Colony_Photos](https://drive.google.com/drive/u/0/folders/1203vZyktuA_R3KwnG9uSV3JXkkiTA-um) and [Acropora_Sampling_Colony_Photos](https://drive.google.com/drive/u/0/folders/19YCW-KpNBIq2hraZCzva5vsIAdVH9cLC) for the September 2022 timepoint.

All timepoint and tracking data sheets for the September 2022 timepoint for both [*Pocillopora*](https://drive.google.com/drive/u/0/folders/1w3KdwBZaaAFh8UdbOXM3KZCofldOC53i) and [*Acropora*](https://drive.google.com/drive/u/0/folders/1YSVr335RNhZXcJyI0TOSr_edmDsTy4zw) were updated.  

### *Spawning*  

We again checked for spawning by stopping the water and pumps in all tanks at 2000 to watch for setting in collected colonies. We did not see setting by 2300. There was no spawning.

# 16 September 2022  

### *Colony collection*    

Parent colonies were collected from the Mahana site on Moorea's north shore (17°29'12.5"S 149°53'16.1"W).

![field site](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/mahana.png)

We first collected n=24 *A. pulchra* experimental colonies for Danielle's project at 0800-0900 at the Mahana field site. We transported these colonies back to Gump station at 1000. Parental colonies had previously experienced a marine heatwave scenario or ambient diel conditions in March-April 2022. All information on the heatwave experiment and data can be found [here](https://github.com/daniellembecker/Gametogenesis/tree/main/heatwave).


We then collected n=25 colonies (approx. 15-30cm in diameter) from large patches of *Acropora pulchra* from the Mahana field site between 1000-1200. We have a year-long project that has been tracking histology and physiology of colonies (Danielle's project) at this site and therefore we have temperature and light data from this site as environmental context. The colonies collected are considered "wildtype" and were not taken from tagged colonies. Colonies were collected from different patches to obtain a genetically diverse sample.

Ariana and Danielle used a hammer and chisel to remove large fragments (15-30cm diameter) from these patches. Ariana Danielle and Pierrick transported them back to the boat where they were kept in coolers and buckets filled with seawater. Colonies were then transported back to Gump station at 1230 and placed in 2 flow through seawater tanks (n=12-13 colonies per tank). Tanks are equipped with 3 small recirculating pumps for water flow and had high water flow.   

![parents tank](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/colonies_tank.jpeg)

These colonies will be used to collect wildtype gametes to rear to larval stages for Ariana's project. Spawn will be collected from each of the 2 tanks with wildtype colonies.  

### *Launching loggers*  

Temperature loggers were launched to track temperature in the adult parent tanks (n=2 tanks for Danielle and n=2 tanks for Ariana; N=4 loggers) and larval conicals (n=4 conicals per blue tank; N=16 loggers).  

Black Onset Hobo loggers were launched in each parent tank at 1900. Pendant Onset Hobo loggers were launched in larval conicals at 2200. Loggers are set to collect data every 10 minutes. At the end of the experiment we will need to do a group calibration to calculate offsets for the loggers.  The loggers and location are as follows:  

| Logger SSN  | Tank |
| ----------- | ----------- |
| 20444040    | DBP Parent 2 |
| 20719656    | Wildtype Parent 2 |
| 20444033    | Wildtype Parent 1 |
| 20946644    | Wildtype Parent 1 |
| 21002982    | Conical A1 |
| 21002974    | Conical A2 |
| 21002979    | Conical A3 |
| 21002972    | Conical A4 |
| 21002981    | Conical B1 |
| 21335980    | Conical B2 |
| 21335987    | Conical B3 |
| 20937873    | Conical B4 |
| 21002977    | Conical C1 |
| 21335982    | Conical C2 |
| 20937872    | Conical C3 |
| 21002980    | Conical C4 |
| 21335986    | Conical D1 |
| 21335983    | Conical D2 |
| 21002975    | Conical D3 |
| 21335984    | Conical D4 |  

Loggers were placed at the bottom of tanks. Pendants were positioned upright to measure light as well as temperature.  

### *Apex system*

Apex temperature probes were calibrated at 2230 using a digital thermometer (Traceable). Temperatures were between 26.1 and 26.4 for all tanks at the time of calibration.    

### *Tank set up*  

We have been having problems with the water flow not keeping up with demand in the spawning systems with low flow coming from the cooler float valve. Ariana fixed this by cleaning the float valves to remove shell fragments and cutting tubing to go straight from the outlet to the float valve with no slack. To keep the system clean, we will purge the line before starting water flow to the coolers and will regularly check the float valves for shells or debris.  

Tanks are now set up and running! We will add temperature control in the next few days.   

### *Spawning*  

Spawning equipment and workflows were set up. Danielle prepared supplies and equipment to measure eggs per bundle, take samples of sperm and eggs for molecular responses, measure fecundity on sample fragments, and fertilize by parent treatment history. See Danielle's [notebook](https://daniellembecker.github.io/DanielleBecker_Lab_Notebook/) and the [E5 public notebook](https://urol-e5.github.io/) for more information on her project.

Ariana gathered supplies needed for wildtype spawning. The protocol for Ariana's spawning work will be as follows:  

See a detailed post on this [protocol here](https://ahuffmyer.github.io/ASH_Putnam_Lab_Notebook/Montipora-capitata-spawning-and-larval-rearing-protocol/).

1. Use gravy separators and mesh bottom cups to collect floating bundles from each of the large parent tanks. Count the number of parents that released to contribute to the gamete pool. Gather gametes into a bucket/bowl.
2. Once all bundles are released, pool all gametes and gently mix.
3. Alloquot 5 mL of bundles into each of 16-20 50 mL falcon tubes.
4. Gently add 40 mL of filtered (50 um) seawater to each falcon tube.
5. Allow falcon tubes to rest gently sideways with occassional gentle rocking for approx. 1 hour or until bundles are broken apart and eggs and sperm are allowed to mix.
6. Remove the sperm water from the falcon tube using a pipette and add 10 mL of new filtered seawater to rinse the eggs. Remove this water.
7. Pour the eggs into the conicals. Use 1 falcon tube for each conical. Have low water flow in the conicals.
8. Place extra material if available into the 5 gallon buckets with filtered seawater using the same proceedure.
9. Check the concial cultures every hour. Remove dead material with a pipette, gently mix, and use kim wipes to remove lipid films.  
10. Gently increase the water flow the next morning and continue hourly checks and cleaning.  

We stopped the water and pumps in all tanks at 2000 to watch for setting in collected colonies. We did not see setting by 2300. There was no spawning. Red lights only were used to look at colonies.   

# 15 September 2022

### *Set up*  

Today we continued setting up tanks and unpacking and organizing supplies. We continued assembling portable spawning systems in preparation for spawning. We made and purchased supplies for banjo filters for larval tanks and Gump staff assisted us with installing filtration systems for our water lines.

This is what our spawninig sytems look like and a full protocol is [available here](https://docs.google.com/spreadsheets/d/1IPcQGoKwVJidsff5xrbiKTevRrtkKmqbUoALj0bd230/edit#gid=0). These systems (pictured below) contain a cooler that acts as a header tank and can be controlled with heaters in combination with a Neptune Apex system. We will set this up in the comiing days. The water in the cooler is sourced from a seawater output. The water in the cooler is then pumped out to a PVC manifold that then delivers water to each larval tank. The larval tanks (i.e., conicals or "squaricals") each have a banjo filter to allow for low thorugh seawater with large surface area to prevent embryos being damaged or lost on the drain. We set up four of these systems this week.   

![Tanks1](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/tanks.jpeg)

![Tanks2](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/conicals.jpeg)

![Filters](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/filters.jpeg)

We set up 4 portable spawning systems each with a cooler and 6 larval conicals in a blue tank. Two of these tanks are for Ariana (labelled A and B) and two are for Danielle (labelled C and D). This provides each of us with a high and ambient temperature system. Tanks A and C will be the high temperature tanks and tanks B and D will be ambient temperature.   

We also talked through plans for spawning and discussed potential protocols for measuring reproductive metrics for Danielle.

Finally, we prepared 6 extra buckets in a blue tank as extra space in case we have more gametes than can fit in the conicals. These buckets will contain seawater and sit in a water bath. They will not be flow through and will require water changes during embryo rearing.  

### *Apex system*

Ariana set up two units of Apex systems in preparation for temperature control in larval conicals. There is one Apex unit for Ariana and one unit for Danielle. This will allow for temperature monitoring in n=3 in each tank (A, B, C, D). The temperature of the conicals will be controlled based on the temperature readings in one high temperature conical. Each Apex system consists of the following parts:

1 Energy Bar AB8
1 Apex control unit
5 PM1 modules
6 Aquabus cables
6 Apex temperature probes

Apex units were connected to WiFi and created virtual outlets for each stage of temperature control. The Apex format looks like this:  

![Filters](https://raw.githubusercontent.com/urol-e5/urol-e5.github.io/master/images/Moorea2022/apex_example.png)

We have the Apex systems named "AH Moorea Sq" and "DBP Moorea Sq".  

# 14 September 2022  

### *Setup*  

Today we arrived in Moorea, French Polynesia. Goals for the day were to unpack supplies, assess space and materials and start experimental set up.

We will be constructing portable spawning systems developed in the Putnam Lab by Ariana Huffmyer and Jill Ashey. A full protocol for these systems and parts list is [available here](https://docs.google.com/spreadsheets/d/1IPcQGoKwVJidsff5xrbiKTevRrtkKmqbUoALj0bd230/edit#gid=0).  

To prepare for spawning we cleaned and prepared tanks and started seawater flow. We will be using 4 large tanks to hold adult/parent colonies and 4 smaller tanks to hold the spawning systems. Ariana and Pierrick cleaned and filled tanks and constructed drain PVC systems for each tank. Danielle set up 24 twenty-two gallon tanks with designated labeling for the parental colonies that had previous experienced a diel marine heatwave, or ambient treatment in March-April 2022. Helped gather all supplies from around the station and organize lab space for spawning and subsequent sampling.
